bin/release

```diff
@@ -19,6 +19,8 @@ rake release
 
 docker build --rm -t codeclimate/codeclimate .
 docker push codeclimate/codeclimate
+docker tag codeclimate/codeclimate "codeclimate/codeclimate:$version"
+docker push "codeclimate/codeclimate:$version"
```

Comment:
Sounds good to me. I don't think that's the script's current behavior so we should probably qualify the first `docker push` as well:

```bash
docker push codeclimate/codeclimate:latest
```